modules/theme/src/themes/default/elements/segment.variables

```diff
@@ -99,3 +99,4 @@
 @emphasizedSegmentBackgroundColor: @lightPageBackground;
 @emphasizedSegmentBorder: @defaultBorderWidth solid @lightBorderColor;
 @emphasizedSegmentBorderRadius: @defaultBorderRadius;
+@emphasizedSegmentPadding: 35px;
```

Comment:
Please add the new line at the end of the file.

Reply:
Thanks!! This has been resolved in commit 7fb25bb